creating_images/guidelines.adoc

```diff
@@ -299,44 +299,27 @@ privileged user.
 Because the user ID of the container is generated dynamically, it will not have
 an associated entry in *_/etc/passwd_*. This can cause problems for applications
 that expect to be able to look up their user ID. One way to address this problem
-is to use link:https://cwrap.org/nss_wrapper.html[nss wrapper] and dynamically
-create a *_passwd_* file with the container's user ID as part of the image's
-start script:
+is to dynamically create a *_passwd_* file with the container's user ID as part
+of the image's `ENTRYPOINT` script. This what a Dockerfile might include:
 
 ----
-export USER_ID=$(id -u)
-export GROUP_ID=$(id -g)
-envsubst < ${HOME}/passwd.template > /tmp/passwd
-export LD_PRELOAD=/usr/lib64/libnss_wrapper.so
-export NSS_WRAPPER_PASSWD=/tmp/passwd
-export NSS_WRAPPER_GROUP=/etc/group
+ENV USER_NAME=nobody \
+    USER_UID=99
+RUN chown -R ${USER_UID}:0 /some/directory \
+    && chmod -R g=u /some/directory /etc/passwd
```

Comment:
this chown isn't really helpful. the group change part is, but making uid 99 own everything isn't.